### DISCUSSION TITLE
Failed to SIMULATE

### DISCUSSION THREAD

**Discussion by lht011129** [NASA-SW-VnV/fret#80] [Q&A] [answered]
I have added PATH environment variables as showed in the fisrt figure. But when I tried to execute the "SIMULATE" I failed, as showed in the second figure. How can I handle this question?
![fig1_path](https://github.com/NASA-SW-VnV/fret/assets/84820163/dceb6cdb-6c2f-4b34-b193-c919c7d51ffe)
![fig2_simulate](https://github.com/NASA-SW-VnV/fret/assets/84820163/04204870-88b8-45ed-802a-f2ed3944839b)

**Comment by andreaskatis** [comment, marked as answer]
Hi @lht011129 ,

Thank you for trying out FRET! The SIMULATE button is disabled because FRET checks whether NuSMV is installed, by checking the output of a call to a `nusmv` command. Our next release, which is coming up pretty soon, addresses this in a more robust way. In the meantime, you have two ways to resolve this locally:

1. Rename the `NuSMV` binary to `nusmv`, that is:

```
mv /home/szy/Desktop/NuSMV-2.6.0-Linux/bin/NuSMV /home/szy/Desktop/NuSMV-2.6.0-Linux/bin/nusmv
```

2. Create a symbolic link called `nusmv` , pointing to the `NuSMV` binary. You can do so by running

```
ln -s /home/szy/Desktop/NuSMV-2.6.0-Linux/bin/NuSMV /home/szy/Desktop/NuSMV-2.6.0-Linux/bin/nusmv
```

Please don't forget to let us know if the above resolves your issue.

Best Regards,

Andreas

**Comment by lht011129** [comment]
Hi&nbsp;Andreas，


Thank you for your mail! With the ways provided by you, I have resolved my issue. Thank you again!


Best Regards,


Lin




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NASA-SW-VnV/fret"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月13日(星期一) 晚上10:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NASA-SW-VnV/fret] Failed to SIMULATE (Discussion #80)





 
Hi @lht011129 ,
 
Thank you for trying out FRET! The SIMULATE button is disabled because FRET checks whether NuSMV is installed, by checking the output of a call to a nusmv command. Our next release, which is coming up pretty soon, addresses this in a more robust way. In the meantime, you have two ways to resolve this locally:
  
Rename the NuSMV binary to nusmv, that is:
  mv /home/szy/Desktop/NuSMV-2.6.0-Linux/bin/NuSMV /home/szy/Desktop/NuSMV-2.6.0-Linux/bin/nusmv   
Create a symbolic link called nusmv , pointing to the NuSMV binary. You can do so by heading into the NuSMV folder (/home/szy/Desktop/NuSMV-2.6.0-Linux/bin , according to your screenshot) and run:
  ln -s /home/szy/Desktop/NuSMV-2.6.0-Linux/bin/NuSMV /home/szy/Desktop/NuSMV-2.6.0-Linux/bin/nusmv  
Please don't forget to let us know if the above resolves your issue.
 
Best Regards,
 
Andreas
